projects/components/src/action-menu/action-menu.component.ts

```diff
@@ -10,14 +10,16 @@ import { CommonUtil } from '../utils';
 /**
  * Value used for the display configuration of action buttons if no input is provided by the caller
  */
-export const DEFAULT_ACTION_DISPLAY_CONFIG: ActionDisplayConfig = {
-    contextual: {
-        featuredCount: 0,
-        styling: ActionStyling.INLINE,
-        buttonContents: TextIcon.TEXT,
-    },
-    staticActionStyling: ActionStyling.INLINE,
-};
+export function getDefaultActionDisplayConfig(): ActionDisplayConfig {
```

Comment:
Calling code will be simpler if you take in a cfg object and apply the defaults on top of it instead of letting callers do it.

```ts
/**
 * Value used for the display configuration of action buttons if no input is provided by the caller
 */
export function getDefaultActionDisplayConfig(cfg: ActionDisplayConfig = {}): ActionDisplayConfig {
    const defaults =  {
        contextual: {
            featuredCount: 0,
            styling: ActionStyling.INLINE,
            buttonContents: TextIcon.TEXT,
        },
        staticActionStyling: ActionStyling.INLINE,
    };
    return {...defaults, ...cfg};
}
```